modules/rostests/winetests/kernel32/debugger.c

```diff
@@ -970,7 +970,14 @@ static void doChild(int argc, char **argv)
     }
     else if (is_wow64)
     {
+#ifdef __REACTOS__
+        static wchar_t expected_path[MAX_PATH];
+        GetWindowsDirectoryW(expected_path, MAX_PATH);
+        wcscat(expected_path, L"\\sysnative\\oleacc.dll");
```

Comment:
See my previous suggestion in actctx for these 3 lines, since the buffer is static.